.flowconfig

```diff
@@ -1,5 +1,4 @@
 [ignore]
-.*/packages/.*/node_modules/.*
```

Comment:
Why is this necessary?

Reply:
Exactly :P thats why i got rid of it.

In all seriousness, this was a remnant from when we were having flow issues back in the day. That line effectively ignored all flow types in dependencies so it's in our interest to remove that. The build still passes so 👍